src/spag/env.rs

```diff
@@ -102,3 +125,25 @@ pub fn set_nested_value(y: &mut Yaml, keys: &[&str], val: &str) {
         panic!(format!("Failed to set key {:?} in {:?}", key, y));
     }
 }
+
+/// If keys is ["a", "b", "c"], then unset y["a"]["b"]["c"]
+pub fn unset_nested_value(y: &mut Yaml, keys: &[&str]) {
+    if keys.is_empty() {
+        panic!("BUG: No keys given to unset in the environment.");
+    }
+    let key = Yaml::String(keys[0].to_string());
+    if let Yaml::Hash(ref mut h) = *y {
+        h.remove(&key);
+        // if keys.len() == 1 {
+            // h.remove(&key);
+        // } else {
+            // traverse nested dictionaries to find the key
+            // if let None = h.remove(&key) { this isn't right
+            //     unset_nested_value()
+            // }
+            
+        // }
```

Comment:
This would be probably be simpler if we had `fn get_nested_dictionary(...)`. Then we could do something like `get_nested_dictionary(&keys[0..keys.len() - 1]).remove(...)` I think.